### PR TITLE
[docs] add header for file exporter.

### LINKED ISSUE #10265: Add status header for file

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
I add a header for file exporter, pls review :)

Fixes #10265

### CODE REVIEW

**Comment by Frapschen** [comment]
hi, pls review :)

**Review by TylerHelmuth — APPROVED** [review]
/cc @pmm-sumo as code owner

File: exporter/fileexporter/README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # File Exporter
 
-This exporter will write pipeline data to a JSON file. The data is written in
+| Status                   |                        |
+| ------------------------ |------------------------|
+| Stability                | [beta]                 |
```
Comment (pmm-sumo) [inline exporter/fileexporter/README.md:5]:
```suggestion
| Stability                | [alpha]                 |
```

**Review by pmm-sumo — CHANGES_REQUESTED** [review]
I believe we should consider it an `alpha` exporter.

This is hinted by the documentation:

> Please note that there is no guarantee that exact field names will remain stable. This intended for primarily for debugging Collector without setting up backends.

**Comment by Frapschen** [comment]
@pmm-sumo done.

**Review by mx-psi — APPROVED** [review]

**Review by pmm-sumo — APPROVED** [review]

File: exporter/fileexporter/README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # File Exporter
 
-This exporter will write pipeline data to a JSON file. The data is written in
+| Status                   |                        |
+| ------------------------ |------------------------|
+| Stability                | [alpha]                 |
```
Comment (codeboten) [inline exporter/fileexporter/README.md:5]:
```suggestion
| Stability                | [alpha]                |
```

File: exporter/fileexporter/README.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # File Exporter
 
-This exporter will write pipeline data to a JSON file. The data is written in
+| Status                   |                        |
+| ------------------------ |------------------------|
+| Stability                | [alpha]                |
+| Supported pipeline types | traces, metrics, logs  |
+| Distributions            | [contrib]              |
```
Comment (codeboten) [inline exporter/fileexporter/README.md:7]:
```suggestion
| Status                   |                       |
| ------------------------ |-----------------------|
| Stability                | [alpha]               |
| Supported pipeline types | traces, metrics, logs |
| Distributions            | [core], [contrib]     |
```

File: exporter/fileexporter/README.md [hunk 4]
```diff
@@ -24,3 +28,7 @@ exporters:
   file:
     path: ./filename.json
 ```
+
+
+[alpha]:https://github.com/open-telemetry/opentelemetry-collector#alpha
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```
Comment (codeboten) [inline exporter/fileexporter/README.md:34]:
```suggestion
[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
[core]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
```

**Pull request merged**